library/std/src/ffi/os_str.rs

```diff
@@ -1068,8 +1068,31 @@ impl fmt::Debug for OsStr {
 }
 
 impl OsStr {
-    pub(crate) fn display(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fmt::Display::fmt(&self.inner, formatter)
+    #[stable(feature = "os_str_display", since = "1.51.0")]
+    /// Returns an object that implements [`Display`] for safely printing strings that may contain
+    /// non-Unicode data.
+    ///
+    /// See also [`Path::display()`].
```

Comment:
There's not any more information in `Path::display()`'s documentation, right?

Reply:
I guess not, no. Should I remove this?

Reply:
I removed the link.